_assets/stylesheets/pages/_spiritual-outfitters.scss

```diff
@@ -0,0 +1,29 @@
+.hr-primary {
+    border-top-color: #253746;
```

Comment:
Does this hex code need to be hardcoded?

Reply:
It does not, updated with $cr-blue. Thanks!